### PR TITLE
Chain seeders

### PR DESCRIPTION
**Pull request by freekmurze**
littlethingsaddup

### CODE REVIEW

File: database/seeds/DatabaseSeeder.php [hunk 1]
```diff
@@ -11,7 +11,8 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-         $this->call(UsersSeeder::class);
-         $this->call(SampleDataSeeder::class);
+         $this
+             ->call(UsersSeeder::class);
```
Comment (tykus) [inline database/seeds/DatabaseSeeder.php]: image

;)

**Comment by JeffreyWay** [comment]
Actually, you can't chain these. I'll have to revert this.

**Comment by JeffreyWay** [comment]
Fixed in 5041b53ba7eaade9a8d76f11795ada422aac5df7

**Comment by freekmurze** [comment]
You can chain 'm: https://github.com/laravel/framework/blob/5.5/src/Illuminate/Database/Seeder.php#L45

That return statement was only added pretty recently, so it'll probably work after `composer update`. Anyways, there are more important things to focus on than chaining seeders 😄

**Pull request merged**